packages/strapi-hook-bookshelf/lib/buildQuery.js

```diff
@@ -0,0 +1,234 @@
+const _ = require('lodash');
+const pluralize = require('pluralize');
+
+/**
+ * Build filters on a bookshelf query
+ * @param {Object} options - Options
+ * @param {Object} options.model - Bookshelf model
+ * @param {Object} options.filters - Filters params (start, limit, sort, where)
+ */
+const buildQuery = ({ model, filters }) => qb => {
+  if (_.has(filters, 'where') && Array.isArray(filters.where)) {
+    // build path with aliases and return a mapping of the paths with there aliases;
+
+    // build joins
+    buildQueryJoins(qb, { model, whereClauses: filters.where });
+
+    // apply filters
+    filters.where.forEach(({ field, operator, value }) => {
+      const { association, model: associationModel, attributeKey } = getAssociationFromFieldKey(
+        model,
+        field
+      );
+
+      let fieldKey = `${associationModel.collectionName}.${attributeKey}`;
+
+      if (association && attributeKey === field) {
```

Comment:
A ternary condition seems appropriated for this one. You can skip my comment if you don't think so, btw :)